### PR TITLE
Print only styles

### PR DESCRIPTION
**Pull request by tilsammans**
Changed `.print_only` to have single responsibility of display property.
Moved the print-only style out of `body` parent, as the non-print doesn't have it either.
Detect presence of this logo and don't render a dom node if it isn't there.
Move hardcoded logo path into a helper, making it slightly less painful to override.

### CODE REVIEW

**Comment by tilsammans** [comment]
The print logo as it is, is quite confusing and possibly unfinished, but at least this change won't include non-existing images and tightens up the css scopes a bit.

File: app/assets/stylesheets/pageflow/print_view.scss [hunk 1]
```diff
@@ -57,14 +59,12 @@ body {
       }
     }
 
-    .print_image, .print_only {
-      display: block;
+    .print_image {
```
[the remark this answers is not in the document]
Comment (tilsammans) [reply app/assets/stylesheets/pageflow/print_view.scss]: I hear you @houndci-bot

[the remark this answers is not in the document]
Comment (tilsammans) [reply app/assets/stylesheets/pageflow/print_view.scss]: Try this one: `.js .page.invert:first-child .contentWrapper:after` (welcome to my life)

File: app/assets/stylesheets/pageflow/print_view.scss [hunk 2]
```diff
@@ -1,3 +1,12 @@
+.print_only {
+  display: block;
+  page-break-inside: avoid !important;
```
[the remark this answers is not in the document]
Comment (tilsammans) [reply app/assets/stylesheets/pageflow/print_view.scss]: Couldn't agree more @houndci-bot

**Comment by tf** [comment]
Haven't had the time to think this one through. Coming to grips with some of Pageflow's legacy CSS needs more concentration than I can muster right now ;)

**Pull request closed without merging**